felix/bpf-gpl/tc.c

```diff
@@ -92,6 +93,12 @@ static CALI_BPF_INLINE int calico_tc(struct __sk_buff *skb)
 	}
 	__builtin_memset(ctx.state, 0, sizeof(*ctx.state));
 
+	if (!ctx.counters) {
+		CALI_DEBUG("Counters map lookup failed: DROP\n");
+		return TC_ACT_SHOT;
+	}
+	inc_counter(ctx.counters, TOTAL_PKTS);
```

Comment:
```suggestion
	inc(ctx, TOTAL_PKTS)
```

some small macro like that? :thinking:

Reply:
sure, just if you don't mind I'll uppercase like `INC` just to express that it is macro?